src/scripts/modules/featured-books/featured-books.less

```diff
@@ -69,7 +69,33 @@
 
       p.description {
         overflow: hidden;
-        height: 60px;
+        max-height: 60px;
+        text-align: justify;
+      
+        &.extended {
+          max-height: 100%;
+          &.show-ellipsis {
+            &:after {
+              display: none;
+            }
+          }
+        }
+
+        &.show-ellipsis {
+          position: relative;
+          &:after {
+            content: '...';
```

Comment:
The previous plugin used `…` instead of `...`
Not sure it matters, but we'll have to adjust the automated tests to look for `...` instead if we leave this.

Reply:
Changed, thanks